src/write.ts

```diff
@@ -70,6 +70,21 @@ function biggerIsBetter(tool: ToolType): boolean {
             return false;
         case 'catch2':
             return false;
+        case 'roblox':
+            if (bench.type === 'roblox') {
+                switch (bench.unit) {
+                    case RobloxUnit.MS_PER_OPERATION:
+                    case RobloxUnit.EXECUTIONS:
+                    case RobloxUnit.READS:
+                    case RobloxUnit.WRITES:
+                    case RobloxUnit.MISSES_PER_OP:
+                        return false;
+                    case RobloxUnit.OPERATIONS_PER_SECOND:
+                    case RobloxUnit.FRAMES_PER_SECOND:
+                        return true;
+                }
+            }
```

Comment:
@ZoteTheMighty here is the bigger is better or smaller is better distinction based on units